src/script/components/avatar/AvatarImage.test.tsx

```diff
@@ -17,14 +17,12 @@
  *
  */
 
-import {waitFor} from '@testing-library/react';
-import {act} from 'react-dom/test-utils';
+import {waitFor, render} from '@testing-library/react';
```

Comment:
This also fixes this warning
![image](https://user-images.githubusercontent.com/1090716/197821626-59e69a84-8f44-4863-a2bd-400b38d5cfdf.png)